*lib-python.3.test.test_urllib2_localnet.BasicAuthTests.test_basic_auth_httperror
*lib-python.3.test.test_urllib2_localnet.BasicAuthTests.test_basic_auth_success
*lib-python.3.test.test_urllib2_localnet.ProxyAuthTests.test_proxy_qop_auth_int_works_or_throws_urlerror
*lib-python.3.test.test_urllib2_localnet.ProxyAuthTests.test_proxy_qop_auth_works
*lib-python.3.test.test_urllib2_localnet.ProxyAuthTests.test_proxy_with_bad_password_raises_httperror
*lib-python.3.test.test_urllib2_localnet.ProxyAuthTests.test_proxy_with_no_password_raises_httperror
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_200
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_200_with_parameters
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_404
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_basic
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_chunked
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_geturl
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_https
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_https_sni
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_https_with_cadefault
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_https_with_cafile
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_info
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_issue16464
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_iteration
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_line_iteration
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_redirection
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_sending_headers
*lib-python.3.test.test_urllib2_localnet.TestUrlopen.test_sending_headers_camel
